### ISSUE TITLE
Value is wrong from api servcie

### ISSUE THREAD

**Issue by hongshibao** [DirectXMan12/k8s-prometheus-adapter#24]
I have a custom metric, which is range [0, 300]. And in prometheus, I see the metric values collected  do be in this range. But I get this value from custom apis apis/custom-metrics.metrics.k8s.io/v1alpha1, and find that this value is larger than 300 (e.g. 1400), which is quite strange.

"metricName":"worker_queue_length_percentage","timestamp":"2017-11-07T10:57:18Z","value":"1600"

The version I used is: k8s 1.7 and k8s-prometheus-adapter v0.1.0. And the custom metric is a pod metric.

**Comment by DirectXMan12** [comment]
You'll have to provide more information.  What object are you fetching the metric on? I'd guess, however, that the value is being aggregated over other labels.  Can you post the full set of labels on the series?  If you query prometheus for `sum(your_metric_name{the_type="some-name"}) by (the_type)`, what gets returned?

**Comment by hongshibao** [comment]
I have found the root cause. In my application, I created serval pods and services. Each pod had a metric called worker_queu_length_percentage. And there could be serval services that selected a same pod. For each service I also created a corresponding service monitor. So in prometheus there were serval records for a pod, with each service having one record. And it seemed that k8s-prometheus-adapter would sum them up for the metric of one pod.

After finding the root cause, I have fixed this issue.
